06_modules_lab/02.py

```diff
@@ -1,4 +1,12 @@
-""" Write a program that reads 2 numbers from sys.argv
-and prints their sum.
-"""
-
+from sys import argv
+
+if len(argv) != 3:
+    print 'Usage:\n%s [a] [b]' %argv[0]
+    exit()
+
+try:
+  a = int(argv[1])
+  b = int(argv[2])
+  print 'The sum of %d and %d is %d.' %(a, b, a+b)
+except ValueError:

```

Comment:
Better to use exception all the way. Specifically, you can remove the first `if` and move all invalid input handling to the except block.
It doesn't matter if a user typed in the wrong number of inputs or their wrong type.